### ISSUE TITLE
Unable to set up with yahoo, codes always wrong.

### ISSUE THREAD

**Issue by nl255** [Rookiestyle/KeePassOTP#142]
## Overview
I am unable to complete the authenticator app setup with Yahoo, it reads the QR code and generates codes fine but every time I try to use one it says invalid code.

## Steps to Reproduce
1. Log into a yahoo account that does not have 2fa set up
2. Go to setup authenticator app, read the QR code, and choose continue
3. Attempt to use the code given to complete the setup.

## Expected Behavior
The code should be accepted.

## Actual Behavior
Yahoo says the code is incorrect.

## Context

Both google and github work fine, only yahoo seems to have the problem.

OS:  Windows
KeePass Version: 2.56
Plugin Version: 1.67

[NOTE]: # (If possible, please attach a d
[Debug_KeePassOTP_20231026T214440Z.txt](https://github.com/Rookiestyle/KeePassOTP/files/13183197/Debug_KeePassOTP_20231026T214440Z.txt)
ebug file. Have a look at the wiki for details.)

**Comment by Rookiestyle** [comment]
I set up a Yahoo account myself and can confirm that the generated codes are not accepted.

In a 2nd step, I tested on my smartphone with Google Authenticator and Authy as those two apps are recommend by Yahoo.
Both apps generate the same codes that KeePassOTP generates and of course they are also not accepted.

Since those two apps generate the same codes, I doubt there is a bug in KeePassOTP - would be the same bug in those two apps as well otherwise.
